packages.md

```diff
@@ -3,35 +3,34 @@
 
 | Package Id     | Readme    | Changelog                 | Published Url       |
 |----------------|-----------|---------------------------|---------------------|
```

Comment:
I'm going to be deleting this entire file so don't worry about it much.

Reply:
Ok

Reply:
Make sure you rebase as I deleted this file and I don't want it to come back.

Reply:
I did, i think I regenerated the file in a recent commit. I just went ahead to delete it again.